ert3/storage/_storage.py

```diff
@@ -35,11 +35,16 @@ def init(*, workspace: Path) -> None:
             workspace=workspace,
             experiment_name=f"{workspace}.{special_key}",
             parameters=[],
+            ensemble_size=-1,
         )
 
 
 def init_experiment(
-    *, workspace: Path, experiment_name: str, parameters: Iterable[str]
+    *,
+    workspace: Path,
+    experiment_name: str,
+    parameters: Iterable[str],
+    ensemble_size: int,
```

Comment:
I'm thinking that perhaps we should revisit the idea of wrapping the implementation in a private method `_init_experiment` that requires `ensemble_size` to be an integer and then have the public `init_experiment` just call `_init_experiment`, but making `init_experiment` requiring `ensemble_size` to be positive 🤔 In that way, we limit the odd hack of not having an ensemble size to within `ert3.storage` (and we can still make it disappear soon).

Reply:
Completely agree, made a suggestion!